.github/workflows/ci-workflow.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install curl git python3 build-essential cmake pkg-config -y
+          sudo apt-get install build-essential cmake libfreetype-dev -y
```

Comment:
Why did you remove dependencies? Could you explain? (the fxcg build is still not working, so testing if it still work with removed stuff is hard)